New Unity Project/Assets/Scripts/Hook.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Hook : MonoBehaviour
+{
+    public Camera mainCamera;
+    public LineRenderer _lineRenderer;
+    public DistanceJoint2D _distanceJoint;
```

Comment:
Do not use publice use [SerializedField] private, also initialize with default value